share/completions/set.fish

```diff
@@ -71,6 +71,7 @@ end
 complete -c set -n "fish_is_nth_token 1" -s e -l erase -d "Erase variable"
 complete -c set -n "fish_is_nth_token 1" -s x -l export -d "Export variable to subprocess"
 complete -c set -n "fish_is_nth_token 1" -s u -l unexport -d "Do not export variable to subprocess"
+complete -c set -n "fish_is_nth_token 1" -s f -l function -d "Make variable scope function"
```

Comment:
"Make variable function-scoped"

Reply:
Agree, I'm just following the existing examples. May as well change other descriptions to use same format.

Reply:
The other examples are adjectives. A variable "is local", a variable "is global".

A variable is not "function".

So the others can stay.

Reply:
Seems like we should not just say "make variable scope universal" so there will be inconsistency between descriptions. That's fine, I reverted other changes.